docs/provision_hosted_cluster_on_mce_local_cluster.md

```diff
@@ -126,104 +123,58 @@ After installing the HyperShift operator and enabling an existing cluster as a h
-      "Name": "clc-dhu-hs1",
-      "baseDomain": "dev09.red-chesterfield.com",
-      "publicZoneID": "Z00953301HRDK0M0YKQGE",
-      "privateZoneID": "Z02815081BZYZVASX5HII",
-      "localZoneID": "Z01446842N9Z8MWQL967F",
-      "proxyAddr": ""
-    }
-    ```
-Specically, we'll need the `publicZoneID`, `privateZoneID`, and `localZoneID`
+That's all! Your hosted cluster is now created.
 
-5. Create the IAM:
+Check the status of your hosted cluster via:
 
     ```bash
-    $ hypershift create iam aws --infra-id $CLUSTER_NAME \
-        --aws-creds $AWS_CREDS \
-        --oidc-storage-provider-s3-bucket-name $BUCKET_NAME \
-        --oidc-storage-provider-s3-region $BUCKET_REGION \
-        --region $REGION \
-        --public-zone-id Z00953301HRDK0M0YKQGE \
-        --private-zone-id Z02815081BZYZVASX5HII \
-        --local-zone-id Z01446842N9Z8MWQL967F \
-        --output-file $IAM_OUTPUT_FILE
+    $ oc get hostedclusters -n local-cluster
     ```
 
-6. We can use the `hypershift create cluster aws` command to create our hosted cluster. If you created the infrastructure and IAM pieces separately, we can specify them as arguments via `--infra-json` and `--iam-json`:
+## Importing the Hosted cluster into MCE via UI
 
-    ```bash
-    $ hypershift create cluster aws \
-        --name $CLUSTER_NAME \
-        --infra-id $INFRA_ID \
-        --infra-json $INFRA_OUTPUT_FILE \
-        --iam-json $IAM_OUTPUT_FILE \
-        --aws-creds $AWS_CREDS \
-        --pull-secret $PULL_SECRET \
-        --region $REGION \
-        --generate-ssh \
-        --node-pool-replicas 3 \
-        --namespace local-cluster \
-        --render > hosted-cluster-cr-render.yaml
-    ```
+After creating the hosted cluster, the next step is to import it into MCE.
 
-Note: This command will also perform what `hypershift create infra` and `hypershift create iam` does if those fields are not specified, however we'll also need to provide the required arguments for those portions (such as the S3 bucket name)
+You can import do this from the MCE Console UI by going to Infrastructure > Clusters and then clicking the your $CLUSTER_NAME in the cluster list and clicking `Import hosted cluster`.
 
-7. Edit the `hosted-cluster-cr-render.yaml` above by adding this annotation to the HostedCluster CR:
+## Importing the Hosted Cluster into MCE via CLI
 
-    ```yaml
-      annotations:
-        cluster.open-cluster-management.io/managedcluster-name: CLUSTER_NAME
-    ```
-If not included, the name used when importing the cluster is the `InfraID` value.
+After creating the hosted cluster, the next step is to import it into MCE.
+
+### How to name your ManagedCluster
+* The $CLUSTER_NAME listed below will be the registred name of your hosted cluster in ACM/MCE.  
+* When the name is provided in the `HostedCluster` annotation `cluster.open-cluster-management.io/managedcluster-name`, it must match CLUSTER_NAME when creating the `ManagedCluster` resource.
```

Comment:
This is good to have, but won't be required by default at 2.7GA if you want ManagedCluster.Name == HostedCluster.Name

Reply:
Thanks @jnpacker! Will update to remove this part.